src/util/message.cpp

```diff
@@ -14,7 +15,27 @@
 #include <string>
 #include <vector>
 
-const std::string strMessageMagic = "Bitcoin Signed Message:\n";
+static const std::string MESSAGE_MAGIC = "Bitcoin Signed Message:\n";
```

Comment:
Renaming this should probably go in the next commit?

Reply:
Done.